lib/matplotlib/path.py

```diff
@@ -188,6 +188,17 @@ def _fast_from_codes_and_verts(cls, verts, codes, internals_from=None):
             pth._interpolation_steps = 1
         return pth
 
+    @classmethod
+    def _make_closed(cls, vertices):
```

Comment:
```suggestion
    def _create_closed(cls, vertices):
```
To make it clear this makes a new path. (It's conceptually possible to make an existing path closed by adding a closing edge.)

Reply:
sure